README.md

```diff
@@ -19,6 +19,7 @@ The Sparkmagic project includes a set of magics for interactively running Spark
 * Automatic visualization of SQL queries in the PySpark, PySpark3, Spark and SparkR kernels; use an easy visual interface to interactively construct visualizations, no code required
 * Easy access to Spark application information and logs (`%%info` magic)
 * Ability to capture the output of SQL queries as Pandas dataframes to interact with other Python libraries (e.g. matplotlib)
+* Send local files or dataframes to a remote cluster (e.g. sending pretrained local ML model straight to the Spark cluster)
```

Comment:
nice work!! don't forget to update the output of %%help to show the new -o param to %%local and the subsequent change to the README.

Reply:
Actually, there is a new cell magic dedicated to that - `%send_to_spark`

It has been already added to `%help` and there is also an example `.ipynb` usage attached: https://github.com/spaszek/sparkmagic/blob/35619b3e2686d1f007cff0f2bcbc685b532e23d9/examples/Send%2Blocal%2Bdata%2Bto%2BSpark%2Bexample.ipynb